dvc/dependency/__init__.py

```diff
@@ -39,13 +33,9 @@
     'https': DependencyHTTP,
 }
 
-# We are skipping RemoteHTTP.PARAM_CHECKSUM because is the same as RemoteS3
-SCHEMA = {
-    DependencyBase.PARAM_PATH: str,
-    schema.Optional(RemoteLOCAL.PARAM_CHECKSUM): schema.Or(str, None),
-    schema.Optional(RemoteS3.PARAM_CHECKSUM): schema.Or(str, None),
-    schema.Optional(RemoteHDFS.PARAM_CHECKSUM): schema.Or(str, None),
-}
+SCHEMA = output.SCHEMA.copy()
+del SCHEMA[schema.Optional(OutputBase.PARAM_CACHE)]
```

Comment:
Also, @efiop, maybe a small comment right here explaining the difference between a dependency and an output (thus, deleting `CACHE` and `METRIC` from the schema) would be helpful for someone that's just familiarizing with the codebase.